config/crd/bases/app.kiegroup.org_kogitoruntimes.yaml

```diff
@@ -548,16 +548,16 @@ spec:
                     type: object
                 type: object
               propertiesConfigMap:
-                description: Custom ConfigMap with application.properties file to
-                  be mounted for the Kogito service. The ConfigMap must be created
-                  in the same namespace. Use this property if you need custom properties
-                  to be mounted before the application deployment. If left empty,
+                description: "Custom ConfigMap with application.properties file to
+                  be mounted for the Kogito service. \n The ConfigMap must be created
```

Comment:
is these new lines characters correctly interpreted by console ui?

Reply:
Yes:
![image](https://user-images.githubusercontent.com/4629376/130467661-845c133a-d5a6-4fa1-a850-2aee05a8f6b3.png)